ChattoAdditions/Source/Input/ChatInputBar.swift

```diff
@@ -250,6 +260,11 @@ extension ChatInputBar: UITextViewDelegate {
     }
 
     public func textView(_ textView: UITextView, shouldChangeTextIn nsRange: NSRange, replacementText text: String) -> Bool {
+        if self.enableReturnKeyToSend && text.range(of: "\n") != nil {
```

Comment:
pasting text that contains a new line is allowed.

@dive thanks, perhaps our needs are not the same